guides/v2.1/comp-mgr/patching.md

```diff
@@ -0,0 +1,63 @@
+---
+group: software-update-guide
+title: Applying patches
+functional_areas:
+  - Upgrade
+---
+
+This topic discusses the workflow for applying a patch to your Magento application.
+
+We strongly recommend that patches be tested in a staging/development environment before being deployed to production.
+
+## First: backup your data
+
+We also strongly recommend that you backup your data before any patches are applied.
+To back up your data, please refer to [Back up and roll back the file system][].
+
+## Get the patch
+
+Magento patches can be found in the [Magento Security Center][]. Follow the instructions on the page to download the patch file, depending on your version and installation type.
+
+## How patches work
+
+Patch (or diff) files are text files that note:
+
+* The file(s) to be changed.
+* The line number to begin the change and the number of lines to be changed.
+* The new code to swap in.
+
+When the [patch][] program is run, this file is read in and the specified changes are made to the file(s).
+
+## Using the command line to apply a patch
+
+1. Login to the server as the Magneto user.
+1. Upload the patch files to the `<Magento_root>` folder.
```

Comment:
If this is in a command line environment, what command/tool is used to do this? `scp`? `ftp`?

This should probably be the first step, since the file is not in the server yet. 

The sequence should be:
1. Upload the local file into a specific directory on the server using ____
2. Login to the server and verify the file is in the correct directory

Reply:
Fixed.